### ISSUE TITLE
Snoopy url not reachable for Ubuntu 24.04(noble)

### ISSUE THREAD

**Issue by abhishek-sa1** [a2o/snoopy#280]
## Checklist before starting to submit this bug report

I confirm that:
- [X] I am submitting a bug report! :)
- [ x] I have tested this with the [latest stable Snoopy version](https://github.com/a2o/snoopy/releases/latest) (or the latest master build).
- [x ] I have checked the [FAQ](https://github.com/a2o/snoopy/blob/master/doc/FAQ.md).
- [ x] I have read Snoopy's documentation [here](https://github.com/a2o/snoopy/blob/master/README.md) and [here](https://github.com/a2o/snoopy/tree/master/doc).
- [x ] I have searched [Snoopy issues](https://github.com/a2o/snoopy/issues?q=is%3Aissue) for an existing issue that matches my problem, and found none.

## Bug description
`E: The repository 'https://a2o.github.io/snoopy-packages/repo/ubuntu noble Release' does not have a Release file.`

Snoopy url fails with above error in Ubuntu 24.04

## Bug reproduction steps

Followed steps in https://github.com/a2o/snoopy/blob/master/doc/install-from-repo.md#ubuntu

## Expected result

Snoopy url should be reachale in Ubuntu 24.04 

## Actual result

`E: The repository 'https://a2o.github.io/snoopy-packages/repo/ubuntu noble Release' does not have a Release file.`

Snoopy url fails with above error in Ubuntu 24.04

**Comment by bostjan** [comment]
Duplicate of #279, I'll do the release soon.

**Comment by bostjan** [comment]
@abhishek-sa1, Snoopy Logger package for Ubuntu 24.04 is now available.
